### DISCUSSION TITLE
Installing from sctracth micropython-ulab to Raspberry Pi Pico

### DISCUSSION THREAD

**Discussion by muhammedAzadKilicarslan** [v923z/micropython-ulab#606] [Q&A]
Hello bro, 

First of all, congratulations for this usefull project. I want to confess something to you, I could not install this library to my pico and I know that there are some people in same situation as me. There is a offer for us. Can you explain how to install this library to pico from sctracth? If you explain detailed step by step, I can make a video on Youtube and metion about your profile and your project. Finally, your project also will be spread.

What do you think?

**Comment by v923z** [comment]
Thanks for your comments! 

Is the section https://github.com/v923z/micropython-ulab#rp2-based-boards not detailed enough? Also, if you don't want to deal with the installation and compilation, you can download the latest firmware from https://github.com/v923z/micropython-builder/releases. If your particular hardware is not listed there, I would be more than happy to add that to the workflow file.

**Comment by muhammedAzadKilicarslan** [comment]
I did all of things in this link "https://github.com/v923z/micropython-ulab#rp2-based-boards" but I do not know what is the next step. If you help me, I can make a video for dumbies as me :| .

**Comment by v923z** [comment]
Ah, I see. So, at the end of those steps, you should have the firmware in the build directory somewhere in `./micropython/ports/rp2/`. You take that firmware, and upload it to the microcontroller. You could look at https://www.youtube.com/watch?v=_JxfOmTY8I4, e.g.

**Comment by muhammedAzadKilicarslan** [comment]
Thank you so much bro. I am really appreciate it.

**Comment by muhammedAzadKilicarslan** [comment]
I tried my best. Regards. https://www.youtube.com/watch?v=LngkFzQla-k&ab_channel=MuhammedAzadK%C4%B1l%C4%B1%C3%A7arslan

**Comment by v923z** [comment]
Thanks for sharing the video!
